src/api/api.service.ts

```diff
@@ -422,4 +422,38 @@ export class ApiService {
       ); 
     }
   }
+
+   async getAllApiContributors(apiId: string){
+    try {
+      //Check if Api exists
+      const api = await this.apiRepo.findOne({where:{id:apiId}})
+      if(!api){
+        throw new NotFoundException(
+          ZaLaResponse.NotFoundRequest(
+            "Not Found Error",
+            "Api Not Found",
+            "404"
+          )
+        )
+      }
+      const contributors = []
+      const contributorIds = api.contributors
+      for(const id of contributorIds){
+        const contributor = await this.profileRepo.findOneBy({id})
+        if(contributor){
+          contributors.push(contributor)
+        }
+      }
+      return contributors
```

Comment:
I would do this instead 
![image](https://user-images.githubusercontent.com/84224449/201221072-5cbc0923-d1ab-4607-97af-bc1301daec45.png)